pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -57,7 +58,8 @@ type MigPlanSpec struct {
 // MigPlanStatus defines the observed state of MigPlan
 type MigPlanStatus struct {
 	Conditions
-	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	Incompatible
+	ObservedGeneration int64 `json:"observedGeneration"`
```

Comment:
This change ^ is reverting https://github.com/konveyor/mig-controller/pull/422.  The `observedGeneration` needs to be `omitempty`.

Reply:
Oh, ok, I see. This resolves the issue, thanks @jortel